src/gluonts/shell/train.py

```diff
@@ -90,6 +91,8 @@ def run_train(
         if "num_workers" in hyperparameters.keys()
         else None
     )
+    if num_workers:
+        multiprocessing.set_start_method("spawn", force=True)
```

Comment:
Maybe minor, but still: this `set_start_method` could be invoked at the beginning of the module, right after the imports, regardless of the number of workers. (If no workers are used, then no subprocess will be launched, so it doesn't matter what start method is being used)

Reply:
True, I moved it!